deep-security-agent/defaults/main.yml

```diff
@@ -0,0 +1,41 @@
+# Copyright:: 2018, Trend Micro
+#
+# License as per [repo](master/LICENSE)
+#
+
+#######################
+# Mandatory Variables #
+#######################
+---
+# As you add agents to Deep Security, they are automatically made available for download through
+#   the same hostname:port as the Deep Security Manager admin interface. For Marketplace and software deployments, pass
+#   this data through the 'dsm_agent_download_hostname' & 'dsm_agent_download_port' attributes.
+dsa_download_hostname: app.deepsecurity.trendmicro.com
+dsa_download_port: 443
+
+# Deep Security agents activate on a different port than the admin interface. For Marketplace and
+#    software deployments, 'dsm_agent_activation_hostname' is typically the same as 'dsm_agent_download_hostname'
+#    but 'dsm_agent_activation_port' is different than 'dsm_agent_download_port'. For Deep Security
+#    as a Service, it's the reverse to make it easy to configure only one outbound port.
+dsa_activation_hostname: agents.deepsecurity.trendmicro.com
+dsa_activation_port: 443
+
+# For multi-tenant deployments of Deep Security (such as Deep Security as a Service), every tenant/organization
+#    is assigned a unique 'tenant_id' and 'tenant_password' that is only used for agent activation. You can
+#    find this info from the Support/Help > Deployment Scripts menu in the Deep Security Manager. Select any agent
+#    and then enable "Activate Agent automatically after installation". This places the tenant_id and 
+#    tenant_password in the script the wizard is building for you.
+tenant_id:
+token:
+
+
+#######################
+# Optional attributes #
+#######################
+
+# The ID of the policy you want to initialize the agent with. This takes precedence over the 
+#    'policy_name' setting. 
```

Comment:
As we don't provide policy_name now, I think the later part can be removed.
PS. The problem also in chef

Reply:
Removed the description about "policy_name".